src/sentry/notifications/utils/participants.py

```diff
@@ -169,18 +169,13 @@ def get_owners(project: Project, event: Event | None = None) -> Iterable[Team |
 
     else:
         outcome = "match"
-        recipients = ActorTuple.resolve_many(owners)
-
-    if len(recipients) > 1:
-        ownership = ProjectOwnership.get_ownership_cached(project.id)
-        # Used to suppress extra notifications to all project members, only notify the would-be auto-assignee
-        if (
-            ownership
-            and not ownership.fallthrough
-            and not features.has("organizations:notification-all-recipients", project.organization)
-        ):
-            return list(recipients)[-1:]
-
+        matched_recipients = ActorTuple.resolve_many(owners)
+        # Used to suppress extra notifications to all matched owners, only notify the would-be auto-assignee
+        recipients = (
```

Comment:
This ternary is fine but I think it would be clearer to do:
```py
recipients = ActorTuple.resolve_many(owners)
# Used to suppress extra notifications to all matched owners, only notify the would-be auto-assignee
if features.has("organizations:notification-all-recipients", project.organization):
   recipients = recipients[-1:]
```